src/index.html

```diff
@@ -10,7 +10,47 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="dig"
```

Comment:
```suggestion
      data-qa="big"
```
it is will help to foune test images in tests